### PR TITLE
use NUMA_NO_NODE macro and num_online_cpus() instead

### PR DESCRIPTION
**Pull request by yeyunfeng-dev**
use NUMA_NO_NODE macro instead of -1 for numa
use num_online_cpus() instead core_count

### CODE REVIEW

File: cputree.c [hunk 1]
```diff
@@ -45,12 +45,11 @@ GList *cache_domains;
 GList *packages;
 
 int cache_domain_count;
-int core_count;
 
 /* Users want to be able to keep interrupts away from some cpus; store these in a cpumask_t */
 cpumask_t banned_cpus;
 
-cpumask_t cpu_possible_map;
+cpumask_t cpu_online_map;
```
Comment (nhorman) [inline cputree.c]: if you're going to do this, you may as well remove the cpu_possible_map extern and its associated macros.

Comment (yeyunfeng-dev) [reply cputree.c]: cpu_present_map is also unused too, is it need to remove together ?
thanks.

Comment (nhorman) [reply cputree.c]: may as well, sure

Comment (yeyunfeng-dev) [reply cputree.c]: ok, patch has been updated now.

Comment (nhorman) [reply cputree.c]: looks good

**Pull request merged**